### PR TITLE
chore: add flow/output namespace to error logs

### PR DESCRIPTION
**Pull request by aslafy-z**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | N/A
| License         | Apache 2.0

### What's in this PR?

Add Namespace to "output not found" log
 
### Why?

Gives more visibility on "output not found" errors

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)

### CODE REVIEW

File: controllers/logging/logging_controller_test.go [hunk 1]
```diff
@@ -446,7 +446,7 @@ func TestSingleFlowDefaultLoggingRefInvalidOutputRef(t *testing.T) {
 	defer ensureCreated(t, logging)()
 	defer ensureCreated(t, flow)()
 
-	expected := "referenced output not found: test-output-nonexistent"
+	expected := fmt.Sprintf("referenced output not found: %s:test-output-nonexistent", testNamespace)
```
Comment (siliconbrain) [inline controllers/logging/logging_controller_test.go:449]:
```suggestion
	expected := fmt.Sprintf("referenced output not found: %s/test-output-nonexistent", testNamespace)
```

File: pkg/resources/model/system.go [hunk 2]
```diff
@@ -282,12 +282,12 @@ func FlowForFlow(flow v1beta1.Flow, clusterOutputs ClusterOutputs, outputs Outpu
...
 			plugin, err := plugins.CreateOutput(output.Spec, outputID, secrets.OutputSecretLoaderForNamespace(output.Namespace))
 			if err != nil {
-				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %q", outputRef))
+				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %s:%s", output.Namespace, outputRef))
 				continue
 			}
 			allOutputs = append(allOutputs, plugin)
 		} else {
-			errs = errors.Append(errs, errors.Errorf("referenced output not found: %s", outputRef))
+			errs = errors.Append(errs, errors.Errorf("referenced output not found: %s:%s", flow.Namespace, outputRef))
```
Comment (siliconbrain) [inline pkg/resources/model/system.go:290]:
```suggestion
			errs = errors.Append(errs, errors.Errorf("referenced output not found: %s/%s", flow.Namespace, outputRef))
```

File: pkg/resources/model/system.go [hunk 3]
```diff
@@ -282,12 +282,12 @@ func FlowForFlow(flow v1beta1.Flow, clusterOutputs ClusterOutputs, outputs Outpu
 			outputID := fmt.Sprintf("%s:output:%s:%s", flowID, output.Namespace, output.Name)
 			plugin, err := plugins.CreateOutput(output.Spec, outputID, secrets.OutputSecretLoaderForNamespace(output.Namespace))
 			if err != nil {
-				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %q", outputRef))
+				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %s:%s", output.Namespace, outputRef))
```
Comment (siliconbrain) [inline pkg/resources/model/system.go:285]:
```suggestion
				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %s/%s", output.Namespace, output.Name))
```

**Review by siliconbrain — COMMENTED** [review]
I'd rather you used the customary `<namespace>/<name>` format.

File: pkg/resources/model/system.go [hunk 4]
```diff
@@ -282,12 +282,12 @@ func FlowForFlow(flow v1beta1.Flow, clusterOutputs ClusterOutputs, outputs Outpu
...
 			plugin, err := plugins.CreateOutput(output.Spec, outputID, secrets.OutputSecretLoaderForNamespace(output.Namespace))
 			if err != nil {
-				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %q", outputRef))
+				errs = errors.Append(errs, errors.WrapIff(err, "failed to create configured output %s/%s", output.Namespace, output.Name))
 				continue
 			}
 			allOutputs = append(allOutputs, plugin)
 		} else {
-			errs = errors.Append(errs, errors.Errorf("referenced output not found: %s", outputRef))
+			errs = errors.Append(errs, errors.Errorf("referenced output not found: %s/%s", flow.Namespace, outputRef))
```
[the remark this answers is not in the document]
Comment (aslafy-z) [reply pkg/resources/model/system.go:290]: @siliconbrain WDYT?

[the remark this answers is not in the document]
Comment (siliconbrain) [reply pkg/resources/model/system.go:290]: Even better 👍

**Review by siliconbrain — APPROVED** [review]
👍

**Review by ahma — APPROVED** [review]
Thanks!

**Pull request merged**